quizApp.js

```diff
@@ -2,16 +2,105 @@ document.addEventListener("DOMContentLoaded", updateForm);
+audio.preload = "auto";
 
 function updateForm() {
+  startBtn.addEventListener("click", startQuiz);
   form.addEventListener("submit", onClickingSubmit);
   shuffleBtn.addEventListener("click", onClickingShuffle);
   searchBtn.addEventListener("keypress", searchQuestion);
 }
 
+let score1 = 0;
+let score2 = 0;
+
+function addScore(playerBtn, score, scoreInput) {
+  return () => {
+    score += 1;
+    playerBtn.innerText = `${playerBtn.innerText.split(":")[0]} : ${score}`;
+    scoreInput.value = `${score}`;
+    if (score >= 10) {
+      audio.play();
+      const icons = document.querySelectorAll("i");
+      icons.forEach((icon) => icon.classList.add("remove-style"));
+    }
+  };
+}
+
+function startQuiz() {
+  const player1 = document.getElementById("player1");
+  const player2 = document.getElementById("player2");
+  let div = document.getElementById("player-details");
+  div.innerHTML = "";
+
+  if (player1.value === "" || player2.value === "") {
+    const p = document.createElement("p");
+    p.innerText = "Please provide the players name";
+    div.appendChild(p);
+  } else {
+    const player1Container = document.createElement("div");
+    div.appendChild(player1Container);
+    player1Container.className = "player-container";
+    const player1Btn = document.createElement("button");
+    player1Btn.innerText = `${player1.value} : 0`;
+    player1Btn.classList.add("scorebtn-1");
+    player1Container.appendChild(player1Btn);
+    const correctIcon1 = document.createElement("i");
+    correctIcon1.className = `fa fa-check-circle scorebtn-1`;
+    correctIcon1.id = "icon-correct1";
+    player1Container.appendChild(correctIcon1);
+    const wrongIcon1 = document.createElement("i");
+    wrongIcon1.className = "fa fa-times-circle scorebtn-1";
+    wrongIcon1.id = "icon-wrong1";
+    player1Container.appendChild(wrongIcon1);
+    const numInput1 = document.createElement("input");
+    numInput1.type = "number";
+    numInput1.id = "num-input1";
+    player1Container.appendChild(numInput1);
+
+    const player2Container = document.createElement("div");
+    player2Container.className = "player-container";
+    div.appendChild(player2Container);
+    const player2Btn = document.createElement("button");
+    player2Btn.innerText = `${player2.value} : 0`;
+    player2Btn.classList.add("scorebtn-2");
+    player2Container.appendChild(player2Btn);
+    const correctIcon2 = document.createElement("i");
+    correctIcon2.className = "fa fa-check-circle scorebtn-2";
+    correctIcon2.id = "icon-wrong2";
+    player2Container.appendChild(correctIcon2);
+    const wrongIcon2 = document.createElement("i");
+    wrongIcon2.className = "fa fa-times-circle scorebtn-2";
+    wrongIcon2.id = "icon-wrong2";
+    player2Container.appendChild(wrongIcon2);
+    const numInput2 = document.createElement("input");
+    numInput2.type = "number";
+    numInput2.id = "num-input2";
+    player2Container.appendChild(numInput2);
```

Comment:
If you find yourself repeating the same code then you can repeat it by looping. In this case you can loop this section of code and using template literals change the `scorebtn-${index}` and other places you use the index.